src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println(new StringBuilder()

```

Comment:
You don't need to use StringBuilder here. It's more efficient to use simple concatenation because the compiler will handle conversion to StringBuilder.